src/tests/event.endpoint.test.js

```diff
@@ -0,0 +1,147 @@
+        it('GET all event should have a 200 status', (done) => { // <= Pass in done callback
+            chai.request('serverUrl')
+                .get('/event')
+                .end((err, res) => {
+                    expect(res).to.have.status(200);
+                    done();
+            });
+        });
+
+        it('GET all event should have expected structure', (done) => { // <= Pass in done callback
+            chai.request(serverUrl)
+                .get('/event')
+                .end((err, res) => {
+                    expect(res.body.result).to.equal('ok');
+                    expect(res.body.code).to.equal(200);
+                    expect(res.body).to.have.property('data');
+                    done();
+            });
+        });
+    });
+
+    describe('GET event by Id', () => {
+        it('GET event by Id should have a 200 status', (done) => {
+            chai.request(serverUrl)
+                .get('/event/12345')
+                .end((err, res) => {
+                    expect(res).to.have.status(200);
+                    done();
+            });
+        });
+
+        it('GET event by wrong Id should not have a 200 status', (done) => {
+            chai.request(serverUrl)
+                .get('/event/666')
+                .end((err, res) => {
+                    expect(res).to.not.have.status(200);
+                    done();
+            });
+        });
+
+        it('GET event by Id should be a single event', (done) => {
+            chai.request(serverUrl)
+                .get('/event/12345')
+                .end((err, res) => {
+                    expect(res).to.be.an('object');
+                    done();
+            });
+        });
+
+    });
+
+    describe('POST event', () => {
+        it('POST event receives an 201 status', (done) => {
+            chai.request(serverUrl)
+                .post('/event')
+                .send({id: 987654321, title: 'ola k ase', descripcion: 'pir pir pir', fecha: '12/01/3078'})
+                .end((err, res) => {
+                    expect(res).to.have.status(201);
+                    done();
+            });
+        });
+
+        it('POST event return value is the same as the sent', (done) => {
+            const sentData = {id: 987654321, title: 'ola k ase', descripcion: 'pir pir pir', fecha: '12/01/3078'};
+            chai.request(serverUrl)
+                .post('/event')
+                .send(sentData)
+                .end((err, res) => {
+                    expect(res.body.data).to.deep.equals(sentData);
+                    done();
+            });
+        });
+
+    });
+
+    describe('PUT event by Id', () => {
+        it('PUT event receives an 201 status', (done) => {
+            chai.request(serverUrl)
+                .put('/event/12345')
+                .send({id: 987654321, title: 'ola k ase 2', descripcion: '', fecha: '01/11/1234'})
+                .end((err, res) => {
+                    expect(res).to.have.status(201);
```

Comment:
It seems you get a 200 status here. Maybe you should check it begins with '2' to make sure it is OK.